docs/source/appendices/acronyms.rst

```diff
@@ -0,0 +1,96 @@
+..
+  Created by: mike garcia
```

Comment:
I'm not sure why, but when I did a `git mv`, it didn't register this as a preexisting file. It's the same file with a few definitions added.